docs/source/concepts/WorkspaceGroup.rst

```diff
@@ -8,17 +8,36 @@ Workspace Group
 .. contents::
   :local:
 
-A WorkspaceGroup is a group of workspaces.
+A WorkspaceGroup is a group of workspaces. The WorkspaceGroup object does not hold any data itself, but instead holds a list of Workspace objects. They appear as an expandable list of workspaces in the MantidPlot interface (the list of workspaces is also called the ADS or *AnalysisDataService*). Thus, their primary function is to add structure to the ADS and make it more readable.
```

Comment:
I think its worth leaving the reference to the ADS in, as this is user documentation and the term is used throughout the document; it is better to have it defined at the start to avoid confusion for beginners.

Thanks for the point regarding the more important usage of the workspace group, I have made this clear.

I don't think I've claimed that the ADS is **just** a list of workspaces though? Again, my aim here is primarily to make sure users know what ADS means from their perspective (which I assume will be a familiarity with the interface)